--- a/legacy/application/views/templates/header.php
+++ b/legacy/application/views/templates/header.php
@@ -13,10 +13,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <meta charset="utf-8">
-    <title><?php echo $title ?> - Jorani</title>
+    <title><?php echo $title ?> - HAYNE Leave</title>
     <?php echo debugbar_head(); ?>
     <link href="<?php echo base_url(); ?>assets/bootstrap/css/bootstrap.min.css" rel="stylesheet">
     <link href="<?php echo base_url(); ?>assets/bootstrap/css/bootstrap-responsive.min.css" rel="stylesheet">
     <link rel="stylesheet" href="<?php echo base_url(); ?>assets/dist/legacy.css">
+    <link rel="stylesheet" href="<?php echo base_url(); ?>assets/hayne/hayne.css">
     <?php CI_Controller::get_instance()->load->helper('language');
     $this->lang->load('global', $language); ?>
