--- a/legacy/application/views/emails/en/new_user.php
+++ b/legacy/application/views/emails/en/new_user.php
@@ -41,2 +41,2 @@
-    Welcome to Jorani {Firstname} {Lastname}. Please use these credentials to <a href="{BaseURL}">login to the
-        system</a> :
+    Welcome to HAYNE Leave, {Firstname} {Lastname}. Please use these credentials to <a href="{BaseURL}">log in to the
+        system</a>:
@@ -59,4 +59,3 @@
     <?php if ($this->config->item('ldap_enabled') == FALSE) { ?>
-        Once connected, you can change your password, as explained <a
-            href="https://jorani.org/how-to-change-my-password.html" title="Link to documentation" target="_blank">here</a>.
+        Once connected, you can change your password in HAYNE Leave.
     <?php } ?>
