--- a/legacy/application/models/Types_model.php
+++ b/legacy/application/models/Types_model.php
@@ -18,13 +18,13 @@ class Types_model extends CI_Model
     /**
      * Get the list of types or one type
-     * @param int $id optional id of a type
+     * @param int|null $id optional id of a type
      * @return array<string, mixed>|null list of type(s) or NULL if list was not found
      */
-    public function getTypes(int $id = 0): ?array
+    public function getTypes(?int $id = null): ?array
     {
-        if ($id === 0) {
+        if ($id === null) {
             $query = $this->db->get('types');
             return $query->result_array();
         }
