--- a/legacy/application/config/routes.php
+++ b/legacy/application/config/routes.php
@@ -326,6 +326,6 @@ $route['api/authorization/userinfo'] = 'authorization/userinfo';
 
 //_______________________________________________
 //Default controllers
-$route['default_controller'] = 'leaves';
+$route['default_controller'] = 'pages/view';
 $route['404_override'] = 'pages/notfound';
 $route['(:any)'] = 'pages/view/$1';
