--- a/legacy/application/views/templates/header.php
+++ b/legacy/application/views/templates/header.php
@@ -1,1 +1,2 @@
     <link rel="stylesheet" href="<?php echo base_url(); ?>assets/hayne/calendar.css">
+    <link rel="stylesheet" href="<?php echo base_url(); ?>assets/hayne/on-demand.css">
@@ -1,1 +1,2 @@
     <script defer type="text/javascript" src="<?php echo base_url(); ?>assets/hayne/full-day.js"></script>
+    <script defer type="text/javascript" src="<?php echo base_url(); ?>assets/hayne/on-demand.js"></script>
